.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: ruby
+
```

Comment:
We need to pin to the same version we set in https://github.com/dnsimple/dnsimple-support/blob/master/.tool-versions#L1 and update the .tool-versions to inform to update this file as we do in other (private) apps.

Example:

```
# Update also: .travis.yml
ruby 2.6.5
```

Reply:
+1

Reply:
✅ Done